lib/ansible/modules/cloud/scaleway/scaleway_snapshots_facts.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+#
+# Scaleway snapshots facts module
+#
+# Copyright (C) 2018 Online SAS.
+# https://www.scaleway.com
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: scaleway_snapshots_facts
+short_description: Scaleway snapshots facts module
+version_added: "2.7"
+author: Remy Leone (@sieben)
+description:
+    - "This module gather facts about snapshots on Scaleway."
+options:
+
+  oauth_token:
+    description:
+     - Scaleway OAuth token.
+    required: true
+
+  region:
+    description:
+    - Scaleway compute zone
+    required: true
+    choices:
+      - ams1
+      - EMEA-NL-EVS
+      - par1
+      - EMEA-FR-PAR1
+
+  timeout:
+    description:
+    - Timeout for API calls
+    required: false
+    default: 30
```

Comment:
Most modules using public cloud providers offer a `validate_certs`/`validate` option[1][2]. I think the same apply here.

[1] https://docs.ansible.com/ansible/latest/modules/ec2_snapshot_facts_module.html#ec2-snapshot-facts-module
[2] https://docs.ansible.com/ansible/latest/modules/os_volume_snapshot_module.html#os-volume-snapshot-module